project/assets/scripts/controller/add-page/add-page.js

```diff
@@ -0,0 +1,25 @@
+// (function () {
```

Comment:
If no longer needed, let's go ahead and remove.

Reply:
👍